### DISCUSSION TITLE
Monitor stops on when conflict on local file.

### DISCUSSION THREAD

**Discussion by ianmartinezhn** [abraunegg/onedrive#1598] [Q&A] [answered]
Hey Guys, been using the tool for a while now. 

Im trying to use --monitor but it stops out of the blue when a local item has a different modified.

i did the following commands: 
```
onedrive --synchronize --verbose --resync
onedrive --synchronize --verbose
onedrive --display-sync-status --verbose
onedrive --monitor --verbose --verbose
```
attaching information
```
onedrive --display-sync-status --verbose
Using 'user' Config Dir: /home/ian/.config/onedrive
Using 'system' Config Dir: /etc/onedrive
Configuration file successfully loaded
Using logfile dir: /var/log/onedrive/
Initializing the OneDrive API ...
Configuring Global Azure AD Endpoints
Opening the item database ...
All operations will be performed in: /media/ian/DATA3/1Drive
Application version: v2.4.13-9-g7857a50
Account Type: personal
Default Drive ID: ***********
Default Root ID:**************
Remaining Free Space: 390170449909
Fetching details for OneDrive Root
OneDrive Root exists in the database
Initializing the Synchronization Engine ...
No pending remote changes - in sync
```

onedrive --monitor --verbose

* tail -f onedrive.log
```
2021-Aug-11 11:50:47.6282427	[DEBUG] Attempting to calculate local filesystem path for bb4b1b33561ed680 and BB4B1B33561ED680!48639
2021-Aug-11 11:50:47.6288674	Processing Pictures/Álbum de cámara/2020/01/20200106_131048208_iOS.heic
2021-Aug-11 11:50:47.6290383	[DEBUG] skip_file evaluation for: 20200106_131048208_iOS.heic
2021-Aug-11 11:50:47.6364612	The file last modified time has changed
2021-Aug-11 11:50:47.6637624	[DEBUG] The local item has a different modified time 2020-Jan-06 13:10:48Z when compared to database modified time 2021-Jan-22 07:29:02Z
```
then it stops.

Le me know if im doing something wrong or missing the conflict handling.

thanks in advance

**Comment by abraunegg** [comment, marked as answer]
@ianmartinezhn 
This 99% sounds like this: https://github.com/abraunegg/onedrive/blob/master/docs/known-issues.md#application-stops-running-without-any-visible-reason

When ever something like this is popping up here are the common elements:
1. Users are usually running Ubuntu of some sort, or a Ubuntu variant
2. Users are based in Europe, accessing the Eastern European Microsoft Data Centre

Most likely:
* Something on / within your Linux OS is the most likely cause
* Your Internet Router .. plausible, but doubtful this is the cause
* Your ISP .. plausible, but doubtful
* Your network path between you and Microsoft was somehow disrupted .. plausible, but then the reports would be everywhere of an issue 
* A change at Microsoft .. plausible, but then significantly more people would be impacted

Things for you to check:
* Exit code of the application when you are seeing this
* Perform the HTTPS debugging to determine why the exit is occurring
* Analyse your network between you and OneDrive

Things that have resolved this in the past:
* Dont use Ubuntu and use some other distribution that is not Ubuntu or Ubuntu based
  * If you cant do this, and you are using Ubuntu's Firewall system 'ufw' - check that it is not blocking your traffic
* Upgrade your kernel version
* Some folk, when using `onedrive --monitor` have found setting the interval to 200 rather than 300 helps - this indicates that something on the local Linux OS is having an issue with TCP wait states of 5 mins .. 

It must also be noted that I am *never ever* able to reproduce this, nor create / replicate an environment where this issue occurs.

If this was a bug in the application - this would be reported hundreds of times a day with more data points to try and replicate .. as such the only viable items are (as per above):
* Something on / within your Linux OS is the most likely cause
* Your network path between you and Microsoft is somehow being disrupted
